sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2463,6 +2463,187 @@ async Task<Page<DocumentHealthcareResult>> NextPageFunc(string nextLink, int? pa
 
         #endregion
 
+        #region Analyze Operation
+
+        /// <summary>
+        /// <a href="https://aka.ms/tanerpii"/>.
+        /// For a list of languages supported by this operation, see
+        /// <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/language-support"/>.
+        /// For document length limits, maximum batch size, and supported text encoding, see
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits"/>.
+        /// </summary>
+        /// <param name="documents">The documents to analyze.</param>
+        /// <param name="language">The language that the document is written in.</param>
+        /// <param name="options">The additional configurable <see cref="AnalyzeOperationOptions"/> </param>
+        /// <param name="cancellationToken"><see cref="CancellationToken"/></param>
+        /// <exception cref="RequestFailedException">Service returned a non-success
+        /// status code.</exception>
+        public virtual async Task<AnalyzeOperation> StartAnalyzeOperationBatchAsync(IEnumerable<string> documents, string language = default, AnalyzeOperationOptions options = default, CancellationToken cancellationToken = default)
```

Comment:
hmm I just realized that the options can't be default. If the user doesn't specify the task, then there is no point to call this API at all.
We can leave the tasks in the options for now, make them mandatory, and once in the design phase, we need to remove the tasks from the options bag

Reply:
what I mean is the method can't have `...  StartAnalyzeOperationBatchAsync(... AnalyzeOperationOptions options = default...)` . it should be  `...  StartAnalyzeOperationBatchAsync(... AnalyzeOperationOptions options)`

Reply:
this still needs to be addressed

Reply:
this is addressed @maririos, please do another round of check and let me know what else needs to be addressed. Also, make new comments instead of adding to the old ones.